server/testutil/webtester/webtester.go

```diff
@@ -93,6 +97,18 @@ func New(t *testing.T) *WebTester {
 	return wt
 }
 
+// FindWithMaxDelay can be used to increase the timeout for finding elements that are known to load slower
+func (wt *WebTester) FindWithMaxDelay(debugID string, maxWait time.Duration) *Element {
```

Comment:
nit: to avoid confusion since `Find()` uses a CSS selector to locate elements, maybe call it `FindByDebugIDWithMaxDelay` ?